server/build.gradle

```diff
@@ -40,3 +48,11 @@ task runWithTests {
 runWithTests.dependsOn test
 
 test.dependsOn(":client:runClientTests")
+
+// Adds the `lib/client` directory to the classpath that is built
+// in the startup script that gradle builds for us. It appears that
+// whatever we put here has `lib/` prepended before it, which is kinda
+// annoying for the static web artifacts, but this does work.
```

Comment:
That *is* kinda annoying... But hey, it works. I suppose it's still a less-hackey solution than using absolute paths
with no guarantees made about where the application will actually be located.